### PR TITLE
API v2 Slate -> Redoc

### PR DESCRIPTION
**Pull request by teesloane**
This PR proposes the change of tools (slate -> redoc) for our API v2 spec-generated documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,9 +36,25 @@ this folder (in our build process) and integrated into the Jekyll Site. Follow
 the [local development guide](./docs/local-development.md) to get started with
 building the Jekyll site.
 
-### `/src-api` - API V2 Build Tooling
+`/src-api` - API V1 and V2 Build Tooling
```
Comment (thomasr8) [inline README.md:39]:
```suggestion
`/src-api` - API v1.1 and v2 Build Tooling
```

File: README.md [hunk 2]
```diff
@@ -36,9 +36,25 @@ this folder (in our build process) and integrated into the Jekyll Site. Follow
...
 
-This is the build tooling we use for automatically generating documentation for
+Our API documentation source can be found in this folder.
+
+**API v1** is written by hand, and compiled to work with
```
Comment (thomasr8) [inline README.md:43]: Should this be "v1.1"? I'm not sure, so I will defer to you, but just wanted to ask.

File: README.md [hunk 3]
```diff
@@ -36,9 +36,25 @@ this folder (in our build process) and integrated into the Jekyll Site. Follow
...
+[Slate](https://github.com/slatedocs/slate). The compilation and deployment of
+`v1` is handled by our `.circleci/config.yml`, which calls our `build_api_docs`
+script. If you need to make changes to our V1 documentation, go to
+`src-api/source/includes` and make changes as needed in the markdown files.
+
+API v2 is compiled from an [OpenAPI
+spec](https://github.com/OAI/OpenAPI-Specification). We use
+[Redoc](https://github.com/Redocly/redoc) to compile our spec into a webpage. To
+see the compilation process, refer to `build_api_docs.sh` and our
+`.circleci/config.yml`. If you need to make changes to the output site, it is
```
Comment (thomasr8) [inline README.md:53]: Maybe use "you will likely" here instead of "it is"?

File: README.md [hunk 4]
```diff
@@ -36,9 +36,25 @@ this folder (in our build process) and integrated into the Jekyll Site. Follow
...
+likely you will need to make source code changes to the API, where the docs are
```
Comment (thomasr8) [inline README.md:54]:
```suggestion
need to make source code changes to the API, where the docs are
```

File: jekyll/_cci2/api-developers-guide.md [hunk 5]
```diff
@@ -260,7 +260,10 @@ The following section details the steps you would need, from start to finish, to
     }' https://circleci.com/api/v2/project/{project-slug}/pipeline
     ```
 
-This concludes the end-to-end example of using the V2 API. For more detailed information about other endpoints you may wish to call, please refer to the [CircleCI API v2 Documentation]({{site.baseurl}}/api/v2/#circleci-api) for an overview of all endpoints currently available.
+This concludes the end-to-end example of using the V2 API. For more detailed
```
Comment (thomasr8) [inline jekyll/_cci2/api-developers-guide.md:263]:
```suggestion
This concludes the end-to-end example of using the v2 API. For more detailed
```

File: jekyll/_cci2/api-intro.md [hunk 6]
```diff
@@ -71,7 +71,7 @@ In the above example the `project_slug` would take the form `:vcs/:org/:project`
 
 The CircleCI API v2 release includes several new endpoints, and deprecates some others. The sections below list the endpoints added for this release, in addition to the endpoints that have been removed.
 
-For a complete list of all API v2 enpoints, please refer to the [API v2 Reference Guide](https://circleci.com/docs/api/v2/#circleci-api), which contains a detailed description of each individual endpoint, as well as information on required and optional parameters, HTTP status and error codes, and code samples you may use in your workflows.
+For a complete list of all API v2 enpoints, please refer to the [API v2 Reference Guide](https://circleci.com/docs/api/v2/), which contains a detailed description of each individual endpoint, as well as information on required and optional parameters, HTTP status and error codes, and code samples you may use in your workflows.
```
Comment (thomasr8) [inline jekyll/_cci2/api-intro.md:74]:
```suggestion
For a complete list of all API v2 endpoints, please refer to the [API v2 Reference Guide](https://circleci.com/docs/api/v2/), which contains a detailed description of each individual endpoint, as well as information on required and optional parameters, HTTP status and error codes, and code samples you may use in your workflows.
```

File: jekyll/_cci2/api-job-trigger.md [hunk 7]
```diff
@@ -12,7 +12,7 @@ This document describes how to trigger jobs using the CircleCI API.
 
 <div class="alert alert-warning" role="alert">
   <p><span style="font-size: 115%; font-weight: bold;">⚠️ Heads up!</span></p>
-  <span> This document refers to using the legacy CircleCI API 1.0, a service that will be eventually be deprecated in favour of the <a href="https://circleci.com/docs/api/v2/#circleci-api">V2 API</a>. Consider using the <a href="https://circleci.com/docs/api/v2/#trigger-a-new-pipeline">Pipelines</a> endpoints to trigger pipelines.</span>
+  <span> This document refers to using the legacy CircleCI API 1.0, a service that will be eventually be deprecated in favour of the <a href="https://circleci.com/docs/api/v2/">V2 API</a>. Consider using the <a href="https://circleci.com/docs/api/v2/#trigger-a-new-pipeline">Pipelines</a> endpoints to trigger pipelines.</span>
```
Comment (thomasr8) [inline jekyll/_cci2/api-job-trigger.md:15]:
```suggestion
  <span> This document refers to using the legacy CircleCI API 1.0, a service that will be eventually be deprecated in favor of the <a href="https://circleci.com/docs/api/v2/">V2 API</a>. Consider using the <a href="https://circleci.com/docs/api/v2/#trigger-a-new-pipeline">Pipelines</a> endpoints to trigger pipelines.</span>
```

**Review by thomasr8 — COMMENTED** [review]
Just some minor comments. Looks really good overall 👍

File: README.md (hunk 2 above)
Comment (teesloane) [reply README.md:43]: I suppose it could be `1.x` but 1.1 works too.

**Review by rosieyohannan — APPROVED** [review]

**Pull request merged**